ChangeLog.md

```diff
@@ -71,6 +75,10 @@
   
   * Change `rpy2` version from 2.9.5 to 3.2.0 (James Balamuta).
 
+  * Change documentation examples to be self-contained (James Balamuta).
```

Comment:
We don't have to do this right away, but I was wondering whether we should break up `elements.md` into a separate file per element? It's a monster page now, and will only be getting longer with time. The `pl-drawing` element is leading the way with the page-per-element strategy. I was thinking that we could make a new top-level category ("Element reference"?).